numberpad.py

```diff
@@ -1887,6 +2104,16 @@ def check_config_values_changes():
 t = threading.Thread(target=check_config_values_changes)
 threads.append(t)
 
+display_wayland_var = os.environ.get('DISPLAY_WAYLAND')
```

Comment:
```suggestion
display_wayland_var = os.environ.get('WAYLAND_DISPLAY')
```
Correct the var name